packages/design-system/src/components/N8nFormInputs/FormInputs.vue

```diff
@@ -19,7 +19,7 @@ type Value = string | number | boolean | null | undefined;
 
 const props = withDefaults(defineProps<FormInputsProps>(), {
 	inputs: () => [],
-	eventBus: createEventBus,
+	eventBus: createFormEventBus,
```

Comment:
This might look more consistent written as `() => createEventBus<FormEventBusEvents>()`

Reply:
That's an unnecessary function wrapping. `createFormEventBus` is already a function, so there's no need to wrap it with a function that only calls the other function